cloud_governance/policy/policy_operations/aws/tag_cluster/tag_cluster_resouces.py

```diff
@@ -228,13 +228,13 @@ def __validate_existing_tag(self, tags: list):
         @return:
         """
         check_tags = ['User', 'Project', 'Manager', 'Owner', 'Email']
```

Comment:
Is it the only place we use it, if not I think we should make a global parameter called:
MANDATORY_TAGS = ['User', 'Project', 'Manager', 'Owner', 'Email']